testcontainers/src/clients/http.rs

```diff
@@ -181,102 +242,22 @@ impl Http {
         };
 
         #[cfg(feature = "watchdog")]
-        if self.inner.command == env::Command::Remove {
+        if client.command == env::Command::Remove {
             crate::watchdog::register(container_id.clone());
         }
 
-        self.inner
+        client
             .bollard
             .start_container::<String>(&container_id, None)
             .await
             .unwrap();
 
-        let client = Http {
-            inner: self.inner.clone(),
-        };
-
-        ContainerAsync::new(container_id, client, image, self.inner.command).await
-    }
-}
-
-impl Http {
```

Comment:
Same as above, it would be nice to not move this code block but keep the diff as minimal as possible :)

Reply:
It should be better now. 👍